packages/context/src/inject-config.ts

```diff
@@ -4,13 +4,27 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {BindingFilter} from './binding-filter';
-import {BindingKey} from './binding-key';
+import {BindingAddress, BindingKey} from './binding-key';
 import {Context} from './context';
 import {ContextView} from './context-view';
 import {assertTargetType, inject, Injection, InjectionMetadata} from './inject';
 import {ResolutionSession} from './resolution-session';
 import {getDeepProperty, ValueOrPromise} from './value-promise';
 
+/**
+ * Injection metadata for `@config.*`
+ */
+export interface ConfigInjectionMetadata extends InjectionMetadata {
+  /**
+   * Path to retrieve the configuration of the target binding
```

Comment:
I find this description a little bit cryptic. IIUC, this path allows users to receive a (deep) property of the configuration object, is that correct? Let's make that more clear please.

I think the name `configPath` may not be very descriptive either. Can we find a better one, e.g. `propertyPath`?  We are already using the name `propertyPath` in `BindingKey` class, let's be consistent.

https://github.com/strongloop/loopback-next/blob/4c5cacafb0d66c99da3760cbf2b9083e6be0ddb6/packages/context/src/binding-key.ts#L41-L44